lib/plant.py

```diff
@@ -1,4 +1,23 @@
+import time
+import board
+import busio
+from adafruit_seesaw.seesaw import Seesaw
+
 class Plant():
 
     def soil_moisture(self):
-        return 350
+
+        i2c_bus = busio.I2C(board.SCL, board.SDA)
+
+        ss = Seesaw(i2c_bus, addr=0x36)
+
+        while True:
+            moisture_level = ss.moisture_read()
+            # temp = ss.get_temp()
+            try:
+                f = open("moisture_readings.txt", "a")
+                f.write("moisture: " + str(moisture_level) + "\n")
+            finally:
+                f.close()	
+            print("moisture: " + str(moisture_level))
```

Comment:
Can str(moisture_level) be returned to the controller, as this would be useful for if statements for calling the Light methods

Reply:
Yes it can, but equally just as an integer reading